enterprise-suite/es-grafana/jvm-exporter.json

```diff
@@ -75,37 +75,35 @@
     "graphName": "GC Rate",
     "promQL": [
       {
-        "expr": "sum by (kubernetes_pod_name)(rate(jvm_PS_MarkSweep_count{ContextTags}[5m]))",
-        "legendFormat": "PS_MarkSweep Rate {{kubernetes_pod_name}}"
-      },
-      {
-        "expr": "sum by (kubernetes_pod_name)(rate(jvm_PS_Scavenge_count{ContextTags}[5m]))",
-        "legendFormat": "PS_Scavenge Rate {{kubernetes_pod_name}}"
+        "expr": "rate( (jvm_gc_collection_seconds_count{ContextTags} or on (kubernetes_pod_name, gc) label_replace({ContextTags,__name__=~\"jvm_.*_count\",gc=\"\"}, \"gc\", \"$1\", \"__name__\", \"jvm_(.*)_count\")) [5m:])",
+        "legendFormat": "{{gc}} rate {{kubernetes_pod_name}}"
```

Comment:
Is `rate` really the best thing here? I think it will be quite misleading - you'll see "1 gc per second" which is not accurate. Instead I'd want to see the number of GCs in the last scrape period. I think increase would be better to be honest, or subtract against offset.

At minimum this should be titled "GC Rate (avg per second)".

Reply:
I'm not sure how top implement subtract against offset, and `increase` is sugar for `rate` so that has the same graph but different y-axis which I don't think you meant.  So I've changed the title. Happy to do some different graph function if it works, but the promql set for counters is limited and I don't know a better way here than rate.

Reply:
Offset should work, e.g. `jvm_gc_collection_seconds_count - jvm_gc_collection_seconds_count offset 30s` <- GC time spent in the last 30s. I guess it will blow up on a counter reset, but this counter should never reset (if pod is redeployed it will be a new time series I think).